Dockerfile

```diff
@@ -27,8 +27,5 @@ RUN rsync -aP rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bedG
 RUN rsync -aP rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bigWigToBedGraph /usr/local/bin
 
 # Download AtacWorks repo
-RUN git clone --recursive -b master https://github.com/clara-genomics/AtacWorks.git
-
-# Install AtacWorks requirements
-RUN pip3 install -r AtacWorks/requirements-base.txt && pip3 install -r AtacWorks/requirements-macs2.txt
-RUN cd AtacWorks && pip3 install --upgrade setuptools && pip3 install .
+RUN pip3 install -i https://test.pypi.org/simple/  --extra-index-url https://pypi.org/simple atacworks==0.2.2
```

Comment:
what's the reason we want to upload to test PyPI repo instead of production PyPI repo?

Reply:
Good catch. Ideally this should be production pypi. Since we haven't done that yet, I have some test versions on test.pypi server and was experimenting with that. We can change it to production PYPI once we upload it there. Thoughts? 

Also, I am able to succesfully install and run atacworks on my local machine, but encountering some issues when running this Dockerfile. Will only merge once I can run it. Wanted to put this up for visibility and feedback

Reply:
Yeah sounds good. For final users they should pull from official PyPI repo

Reply:
In order not to "burn" the version number in PyPI production repo

Reply:
Done.

Reply:
done.